src/OrleansServiceBus/Providers/Streams/EventHub/CachePressureMonitors/AggregatedCachePressureMonitor.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.ServiceBus.Providers
+{
+    /// <summary>
+    /// Aggregated cache pressure monitor
+    /// </summary>
+    public class AggregatedCachePressureMonitor : List<ICachePressureMonitor>, ICachePressureMonitor
+    {
+        /// <summary>
+        /// Record cache pressure to every monitor in this aggregated cache monitor group
+        /// </summary>
+        /// <param name="cachePressureContribution"></param>
+        public void RecordCachePressureContribution(double cachePressureContribution)
+        {
+            this.ForEach(monitor =>
+            {
+                monitor.RecordCachePressureContribution(cachePressureContribution);
+            });
+        }
+
+        /// <summary>
+        /// Add one monitor to this aggregated cache monitor group
+        /// </summary>
+        /// <param name="monitor"></param>
+        public void AddCachePressureMonitor(ICachePressureMonitor monitor)
+        {
+            this.Add(monitor);
+        }
+
+        /// <summary>
+        /// If any mornitor in this aggregated cache monitor group is under pressure, then return true
+        /// </summary>
+        /// <param name="utcNow"></param>
+        /// <returns></returns>
+        public bool IsUnderPressure(DateTime utcNow)
+        {
+            bool isUnderPressure = false;
+            
+            this.ForEach(monitor =>
```

Comment:
nit:  Link statement would be much cleaner.

return this.Any(monitor => monitor.IsUnderPressure(utcNow);